cypress.config.js

```diff
@@ -1,11 +1,27 @@
-const { defineConfig } = require("cypress");
+const { defineConfig } = require('cypress');
+const { addMatchImageSnapshotPlugin } = require('cypress-image-snapshot/plugin');
+const fs = require('fs');
+const path = require('path');
 
 module.exports = defineConfig({
-  chromeWebSecurity: false,
   e2e: {
-    baseUrl:'https://www.saucedemo.com',
+    defaultCommandTimeout: 10000, 
+    pageLoadTimeout: 60000, 
     setupNodeEvents(on, config) {
-     
+      addMatchImageSnapshotPlugin(on, config);
+
+      on('task', {
+        clearScreenshots() {
+          const screenshotsDir = path.join(config.screenshotsFolder, 'baseline');
+          if (fs.existsSync(screenshotsDir)) {
+            fs.rmSync(screenshotsDir, { recursive: true, force: true });
```

Comment:
You canadjust your clearScreenshots task to conditionally clear the baseline screenshots based on whether it's the first run or a subsequent one:

on('task', {
  clearScreenshots({ baseline }) {
    const screenshotsDir = path.join(config.screenshotsFolder, baseline ? 'baseline' : '');
    if (fs.existsSync(screenshotsDir)) {
      fs.rmSync(screenshotsDir, { recursive: true, force: true });
    }
    fs.mkdirSync(screenshotsDir, { recursive: true });
    return null;
  },
});